Echo.Process/Session/SessionDataTypeResolve.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using LanguageExt;
+using static LanguageExt.Prelude;
+using static Echo.Process;
+
+namespace Echo.Session
+{
+    public static class SessionDataTypeResolve
```

Comment:
This shouldn't be `public`

Reply:
There also seems to be a real mix of `internal` and `private` for the member functions.  This type should be `internal` or `private` and the member functions that need to be accessed externally should be `public`.